src/docs/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -222,17 +222,17 @@ for more explorations on latency and memory usage.
 Letting your application's `UIApplicationDelegate` subclass `FlutterAppDelegate`
 is recommended but not required.
 
-The `FlutterAppDelegate` performs functions such as
+The `FlutterAppDelegate` performs functions such as:
 
 - Forwarding application callbacks such as [`openURL`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application)
   to plugins such as [local_auth](https://pub.dev/packages/local_auth).
 - Forwarding status bar taps (which can only be detected in the AppDelegate) to
   Flutter for scroll-to-top behavior.
 
-If your app delegate cannot directly subclass `FlutterAppDelegate`, you should
+If your app delegate can't directly make `FlutterAppDelegate` a subclass, 
```

Comment:
The app delegate doesn't make a subclass, it should be a subclass of FlutterAppDelegate.  It sounds weird, but "can't directly subclass `FlutterAppDelegate`" is clearer.